src/server.h

```diff
@@ -81,6 +81,13 @@ typedef long long ustime_t; /* microsecond time type. */
 #include "rax.h"        /* Radix tree */
 #include "connection.h" /* Connection abstraction */
 #include "memory_prefetch.h"
+#include "trace/trace.h"
+
+#ifdef USE_LTTNG
+#define zfork() do_fork()
+#else
+#define zfork() fork()
```

Comment:
```suggestion
#define valkey_fork() do_fork()
#else
#define valkey_fork() fork()
```
`z<blah>` was historically reserved for memory stuff. Special implementation details normally get valkey_, valkey_strlcpy.